source/blog/2018-02-16-ember-3-1-beta-released.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Ember 3.1 Beta Released
+responsive: true
+author: Matthew Beale
+tags: Releases
+---
+
+Today we're releasing Ember 3.1-beta.1, including releases of Ember.js, Ember
+Data, and Ember CLI.
+
+Traditionally beta releases share a blog post with the
+corresponding stable release, however to keep our messaging clear during the
+transition we've split them up today. See the [Ember 3.0
+Release Post](/blog/2018/02/14/ember-3-0-released.html) for more details on Ember 3.0.
+
+## Ember.js
+
+---
+
+Ember.js is the core of the Ember framework. It provides routing, rendering, and
+dependency injection features.
+
+### Changes in Ember.js 3.1-beta.1
+
+Ember 3.1-beta is an minor release containing several new features and bug
+fixes. It includes a bump of Glimmer VM, Ember's rendering implementation,
+to version 0.30.5. Per our release cycle, these features will be released as
```

Comment:
I believe the version of Glimmer VM that went to into this release is `0.32.0` per https://github.com/emberjs/ember.js/blob/c0b88bcf26e23225502ab16f992cf3313f010b5c/package.json#L62